src/zimwriterfs/zimwriterfs.cpp

```diff
@@ -283,7 +283,7 @@ void parse_args(int argc, char** argv)
 
   do {
     c = getopt_long(
-        argc, argv, "hVvijxuw:I:t:d:c:l:p:r:e:n:m:J:UBL:", long_options, &option_index);
+        argc, argv, "hVvijxuUBLw:I:t:d:c:l:p:r:e:n:m:J:", long_options, &option_index);
```

Comment:
Why changing this line ?
The `:` after the `L` was about `-L` (`--longDescription`) expecting a value.
If you remove it, getopt will probably not parse the option correctly.

Reply:
Yes, will fix